eve/main.yml

```diff
@@ -126,20 +126,27 @@ stages:
     steps:
       - Git: *clone
       - ShellCommand: *npm-install
+      - ShellCommand: *credentials
       - ShellCommand:
           name: Linting
           command: |
+            set -ex
             npm run --silent lint -- --max-warnings 0
             npm run --silent lint_md
             flake8 $(git ls-files "*.py")
             yamllint $(git ls-files "*.yml")
       - ShellCommand:
           name: Unit Coverage
           command: |
+            set -ex
             mkdir -p $CIRCLE_TEST_REPORTS/unit
+            npm run unit_coverage
```

Comment:
I think `unit_coverage` runs the same tests as `test` and `unit_coverage_legacy_location` runs the same as `test_legacy_location`. Does eve store the test output somewhere as an artifact when using the `unit_coverage` options? If so, we can probably remove `npm run test` and `npm run test_legacy_location`.

Reply:
Sounds good, I'll do that. Thanks!